js/msp/msp_protocol.js

```diff
@@ -0,0 +1,212 @@
+'use strict';
+
+var MSP = {
```

Comment:
Not using the same name for a file and the object it defines is confusing for people new to the code.